pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
I think it can keep to target 1.580 by not using `Symbol` feature in tests.
But I don't think it's a good idea as it should make it difficult to maintain the plugin.